tests/test_operation_logs.py

```diff
@@ -0,0 +1,23 @@
+
+from tests import factories
+from tests.common import ApiBaseTest
+from webservices.rest import api
+from webservices.resources.operations_log import OperationsLogView
+
+class TestOperationsLog(ApiBaseTest):
```

Comment:
@pkfec great work on this! My only thought is - since we're filtering for only `status_num=1` in the API, I think we should add an automated test to make sure other `status_num=1` values are not returned. Thoughts?